Python/Python.sublime-syntax

```diff
@@ -342,34 +519,27 @@ contexts:
             - match: (?=\))
               pop: true
             - include: keyword_arguments
+            - match: ','
+              scope: punctuation.separator.parameters.python
+            - include: inline_for
             - include: expressions
 
   lambda:
-    - match: \b(lambda)(?=\s+)
+    - match: \b(lambda)
       scope: storage.type.function.inline.python
       push:
         - meta_scope: meta.function.inline.python
+        - meta_content_scope: meta.function.inline.parameters.python
+        - include: line_continuation_or_pop
         - match: '\:'
           scope: punctuation.section.function.begin.python
           pop: true
-        - match: \s+
-          push:
-            - meta_content_scope: meta.function.inline.parameters.python
-            - match: (?=\:)
-              pop: true
-            - include: keyword_arguments
-            - match: '({{identifier}})\s*(?:(,)|(?=[\n\)\:]))'
-              captures:
-                1: variable.parameter.python
-                2: punctuation.separator.parameters.python
-
-  late-keywords:
-    - match: \b(def|lambda)\b
-      scope: storage.type.function.python
-    - match: \b(class)\b
-      scope: storage.type.class.python
-    - match: \b(async)\b
```

Comment:
Is this actually desirable? `async` on its own doesn't mean anything, and the lack of coloration could prompt the programmer to continue with `for`, `with`, or `def` (and maybe `yield` in the future...)